frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingStatusCell/helpers.ts

```diff
@@ -9,6 +12,10 @@ import {
 import { OsSettingsTableStatusValue } from "../OSSettingsTableConfig";
 import TooltipInnerContentActionRequired from "./components/Tooltip/ActionRequired";
 
+export const isDiskEncryptionProfile = (profileName: string) => {
+  return profileName === FLEET_FILEVAULT_PROFILE_DISPLAY_NAME;
+};
```

Comment:
heads-up: https://github.com/fleetdm/fleet/pull/18212#discussion_r1562490846

> I just realized that I've didn't remove [dev note](https://www.figma.com/file/CtUf2RRqK4eEOTv6W5YjUD/%2317314-Resend-a-configuration-profile?type=design&node-id=324-522&mode=design&t=CFdeqrUJAZdQ35z5-11) about disk encryption profile. We decided to let users resend the disk encryption profile since it [won't make end-users encrypt again](https://fleetdm.slack.com/archives/C03C41L5YEL/p1711396883606039). I hope this isn't huge change, to remove this limitation?